initialize.js

```diff
@@ -29,7 +30,8 @@ function exe(command) {
 }
 
 function fundAll() {
-  exe(`${cli} keys generate ${process.env.SOROBAN_ACCOUNT}`);
+  exe(`${cli} keys generate ${process.env.SOROBAN_ACCOUNT} | true`);
+  exe(`${cli} keys fund ${process.env.SOROBAN_ACCOUNT}`);
```

Comment:
instead of calling a separate command, we can do `keys generate --fund` instead.